adoc/chapters/programming_interface.adoc

```diff
@@ -91,6 +91,7 @@ For compatibility with SYCL 1.2.1, SYCL provides another standard header file:
 [code]#<sycl/sycl.hpp>#.
 In that case, all SYCL classes, constants, types and functions defined by this
 specification should exist within the [code]#::cl::sycl# {cpp} namespace.
+The [code]#<CL/sycl.hpp># header is deprecated.
```

Comment:
The header is tightly connected to the `cl::sycl::` namespace.  Should we say that the namespace is deprecated too?

Reply:
Good point.  Fixed in 167e295e6dcb6b7fd66cb2916d9d3b5982b6020f.